tests/unit/test_bo.py

```diff
@@ -111,12 +111,21 @@ def test_acquisition():
     acq_noise_cov = np.random.rand(n_params, n_params) * 0.5
     acq_noise_cov += acq_noise_cov.T
     acq_noise_cov += n_params * np.eye(n_params)
-    t = 1
+    with pytest.raises(ValueError):
+        acquisition.LCBSC(target_model, noise_var=acq_noise_cov)
+
+    # check acquisition with negative variances
+    acq_noise_var = -0.1
+    with pytest.raises(ValueError):
+        acquisition.LCBSC(target_model, noise_var=acq_noise_cov)
```

Comment:
we set `acq_noise_var` but test `noise_var=acq_noise_cov`

Reply:
Is it a problem? `acq_noise_var` is the name of the variable?

Reply:
ah i assumed it is a problem that we define a variable and then do not use it. or to be exact i assumed `acq_noise_var` is defined with a negative value so that we could test that an error is raised when we set `noise_var` in the acquisition method this value, but here we do not use the variable`acq_noise_var` at all

Reply:
see what you mean, I was confused about the *`cov` -part (not sure why the test is there. the same thing could be tested with any array > 1D). Anyways, fixed now. Thanks.